inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ReplaceFunction.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sdk.transform.process.function;
+
+import org.apache.inlong.sdk.transform.decode.SourceData;
+import org.apache.inlong.sdk.transform.process.Context;
+import org.apache.inlong.sdk.transform.process.operator.OperatorTools;
+import org.apache.inlong.sdk.transform.process.parser.ValueParser;
+
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.Function;
+
+import java.util.List;
+
+/**
+ * ReplaceFunction
+ * description: replace(s, s1, s2)--replace string s1 in string s with string s2.
+ */
+public class ReplaceFunction implements ValueParser {
+
+    private ValueParser stringParser1;
+    private ValueParser stringParser2;
+    private ValueParser stringParser3;
+
+    /**
+     * Constructor
+     * @param expr
+     */
+    public ReplaceFunction(Function expr) {
+        List<Expression> expressions = expr.getParameters().getExpressions();
+        stringParser1 = OperatorTools.buildParser(expressions.get(0));
+        stringParser2 = OperatorTools.buildParser(expressions.get(1));
+        stringParser3 = OperatorTools.buildParser(expressions.get(2));
+    }
+
+    @Override
+    public Object parse(SourceData sourceData, int rowIndex, Context context) {
+        Object stringObj1 = stringParser1.parse(sourceData, rowIndex, context);
+        Object stringObj2 = stringParser2.parse(sourceData, rowIndex, context);
+        Object stringObj3 = stringParser3.parse(sourceData, rowIndex, context);
+        String str1 = OperatorTools.parseString(stringObj1);
+        String str2 = OperatorTools.parseString(stringObj2);
+        String str3 = OperatorTools.parseString(stringObj3);
+        return str1.replace(str2.charAt(0), str3.charAt(0));
```

Comment:
It's more like replace single char instead of a string

Reply:
Resolved